specification/immunisation-fhir-api.yaml

```diff
@@ -1003,40 +1362,205 @@ components:
+            - Schedule
+            - SearchParameter
+            - ServiceRequest
+            - Slot
+            - Specimen
+            - SpecimenDefinition
+            - StructureDefinition
+            - StructureMap
+            - Subscription
+            - Substance
+            - SubstanceNucleicAcid
+            - SubstancePolymer
+            - SubstanceProtein
+            - SubstanceReferenceInformation
+            - SubstanceSourceMaterial
+            - SubstanceSpecification
+            - SupplyDelivery
+            - SupplyRequest
+            - Task
+            - TerminologyCapabilities
+            - TestReport
+            - TestScript
+            - ValueSet
+            - VerificationResult
+            - VisionPrescription
+        id:
+          type: string
+          pattern: '[A-Za-z0-9\-\.]{1,64}'
+          description: The logical id of the resource, as used in the URL for the resource. Once assigned, this value never changes.
+        meta:
+          $ref: '#/components/schemas/Meta'
+          description: The metadata about the resource. This is content that is maintained by the infrastructure. Changes to the content might not always be associated with version changes to the resource.
+        implicitRules:
+          type: string
+          pattern: \S*
+          description: A reference to a set of rules that were followed when the resource was constructed, and which must be understood when processing the content. Often, this is a reference to an implementation guide that defines the special rules along with other profiles etc.
+        language:
+          type: string
+          pattern: '[^\s]+(\s[^\s]+)*'
+          description: The base language in which the resource is written.
+        identifier:
+          $ref: '#/components/schemas/Identifier'
+          description: A persistent identifier for the bundle that won't change as a bundle is copied from server to server.
+        type:
+          type: string
+          pattern: '[^\s]+(\s[^\s]+)*'
+          description: Indicates the purpose of this bundle &ndash; how it is intended to be used.
+        timestamp:
+          type: string
+          pattern: ([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))
+          description: The date/time that the bundle was assembled &ndash; i.e. when the resources were placed in the bundle.
+        total:
+          type: integer
+          format: int32
+          description: If a set of search matches, this is the total number of entries of type 'match' across all pages in the search.  It does not include search.mode = 'include' or 'outcome' entries and it does not provide a count of the number of entries in the Bundle.
+        link:
+          type: array
+          items:
+            $ref: '#/components/schemas/Bundle_Link'
```

Comment:
Some of these schemas don't seem to exist.

Reply:
Line 1930. It's part of `Bandle_Entry`

Reply:
Ah, my mistake. RapiDoc didn't seem to be able to find it, but it was just being slow.